examples/custom_parse.cpp

```diff
@@ -9,6 +9,7 @@
 #include <CLI/CLI.hpp>
 #include <iostream>
 #include <sstream>
+#include <string>
```

Comment:
Why are these adding string with no changes? IWYU?

Reply:
the update to cpplint includes a lot more IWYU checks so this came up.